source/index.html

```diff
@@ -1,12 +1,147 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class = "page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="../images/logo_sedona.svg" width="185" height="96" alt="Седона">
+      </a>
+      <nav class="main-nav">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a>
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Фото и видео
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Форма отзыва
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="https://htmlacademy.ru/">
+              HTML Academy
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-container">
+      <h1 class="visually-hidden">Расскажем про Седону и причины посетить этот городок!</h1>
+      <img class="main-sedona-pic" src="../images/sedona-1.jpg" alt="Седона фотография на фон" width="1200" height="485">
+      <img class="main-welcome-pic" src="../images/welcome.svg" alt="Добро пожаловать в Седону" width="458" height="352">
+      <section class="advantages">
+        <div class="advantages__wrapper">
+          <header class="advantages__header">
+            <h2 class="visually-hidden">
+              Причины посетить Седону
+            </h2>
+            <p class="advantages__slogan">
+              Седона — небольшой городок в Аризоне, заслуживающий большего!
+            </p>
+            <p class="advantages__intro">
+              Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!
+            </p>
+          </header>
+          <ol class="advantages__list">
+            <li class="advantages__item advantages__item--wrapper">
+              <div class="advantages-with-pic-description">
+                <h3 class="advantages__title white__title">Настоящий <br>городок</h3>
```

Comment:
white__title выпадает из БЭМ. Нигде не создан блок white. Если хочешь выделить цвет текста через БЭМ, то скорее всего нужно так
```suggestion
                <h3 class="advantages__title text text--white">Настоящий <br>городок</h3>
```